templates/sparc/greth/mods.config

```diff
@@ -132,7 +132,8 @@ configuration conf {
 	include embox.framework.LibFramework
 	include embox.arch.sparc.libarch
 	include embox.compat.libc.all
-	include embox.compat.libc.math_openlibm
+	//include embox.compat.libc.math_openlibm
+	include embox.compat.libc.math_builtins
```

Comment:
Something wrong with openlibm for sparc?

Reply:
Yes, openlibm goes not support SPARC. [https://openlibm.org/](https://openlibm.org/)

> OpenLibm also supports arm, aarch64, ppc64le, mips, wasm32, and s390(x).